File: Data Structures & Algorithms/top-k-elements-in-list/submission-1.py
class Solution:
    def topKFrequent(self, nums: List[int], k: int) -> List[int]:
        ele  = list(set(nums))
        ct = [nums.count(i) for i in ele]
        ans = []
        for i in range(k):
            idx = ct.index(max(ct))
            ans.append(ele[idx])
            ele.pop(idx)
            ct.pop(idx)

        return ans

File: Data Structures & Algorithms/remove-element/submission-1.cpp
class Solution {
public:
    int removeElement(vector<int>& nums, int val) {
        int r = 0;
        for(int c = 0;c<nums.size();c++){
            if(nums[c] != val){
                nums[r++] = nums[c];
                
            }
        }
        return r;
    }
};